cterasdk/core/plans.py

```diff
@@ -23,3 +37,82 @@ def get(self, name, include=None):
         if plan.name is None:
             raise CTERAException('Could not find subscription plan', None, name=name)
         return plan
+
+    def add(self, name, retention=None, quotas=None):
+        """
+        Add a subscription plan
+
+        :param dict,optional quotas: The data retention policy
```

Comment:
The doc here is incorrect (missing name and you have quotas twice. 
In addition, I don't like the dictionary parameter. What's the expected structure? Should probably use a class

Reply:
I could implement it in a class. The plan XML is very cumbersome and I wanted to provide a simple interface to the client. A key-value dictionary with quantities. You're probably right. As we allow to customize more attributes, i'd make sense to use a class and simply copy the metadata to the correct format.